### ISSUE TITLE
Options for mounting Webdav / davfs2 - possibly include davfs2 package

### ISSUE THREAD

**Issue by harpingon** [clearlinux/distribution#1450]
Searching swupd for davfs / webdav / davfs2 produces nothing.

What are my options for mounting WebDAV into the filesystem? Fuse3?

Other distros do bundle davfs2 (e.g. davfs2 in Arch)

**Comment by ahkok** [comment]
The request for `davfs2` is entirely reasonable, the problem is that it depends on `neon` which hasn't seen a release in 3+ years. Normally that's kind of an edge case, but, in this case, the last release added some major features including `openssl` support. I do not believe that in the 3 years after this was released there was no significant bugfix, and the neon mailinglist seems to confirm that suspicion.

In other words, it appears `neon` is unmaintained.

And because of that, we can't add `davfs2`. We shouldn't.

**Comment by ahkok** [comment]
A good alternative is e.g. `nextcloud` which has integrated fuse support in clearlinux, btw. If Nextcloud is an option for you.

**Comment by harpingon** [comment]
@ahkok  my nextcloud is out there in the cloud, it was the files from it that I was wanting to mount and access. Not sure how I can leverage nextcloud on Clear to access my Nextcloud in the cloud? Given that we appear to still not have the Gnome shell integration (although, that's not a cure all because things like Blender etc. don't use the Gnome save/load dialogues).

**Comment by ahkok** [comment]
Nextcloud can be setup using gnome online accounts - start gnome settings, open online accounts, add the nextcloud account. In the file manager, you can now see the nextcloud storage, and it's mounted as any normal fuse mount. I use this myself with my personal nextcloud instance (incedentally, I run it on clearlinux as well, but it doesn't have to be).

**Comment by harpingon** [comment]
Oh I see now. @ahkok , but it kind of mounts in an odd place:
`
/run/user/1000/gvfs/dav\:host\=myserver.com\,ssl\=true\,user\=myuser\,prefix\=%2Fremote.php%2Fwebdav/`

I guess I could symlink that. That would be good enough, and would work outside of Nautilus.

Curious which FUSE module is doing this, couldn't see anything. Or is Nautilus/GNOME communicating with FUSE directly.

Oh well, either way. This will work for me, I think. Thanks, this can be closed

**Comment by harpingon** [comment]
Oh I see now. @ahkok , but it kind of mounts in an odd place:
`
/run/user/1000/gvfs/dav\:host\=myserver.com\,ssl\=true\,user\=myuser\,prefix\=%2Fremote.php%2Fwebdav/`

I guess I could symlink that. That would be good enough, and would work outside of Nautilus.

Curious which FUSE module is doing this, couldn't see anything. Or is Nautilus/GNOME communicating with FUSE directly.

Oh well, either way. This will work for me, I think. Thanks, this can be closed

**Comment by harpingon** [comment]
Oh I see now. @ahkok , but it kind of mounts in an odd place:
`
/run/user/1000/gvfs/dav\:host\=myserver.com\,ssl\=true\,user\=myuser\,prefix\=%2Fremote.php%2Fwebdav/`

I guess I could symlink that. That would be good enough, and would work outside of Nautilus.

Curious which FUSE module is doing this, couldn't see anything. Or is Nautilus/GNOME communicating with FUSE directly.

Oh well, either way. This will work for me, I think. Thanks, this can be closed

**Comment by thiagomacieira** [comment]
It's gvfsd.
